File: TiffinBox/app/src/main/java/com/example/tiffinbox/Seller/Model/AddRecipe.java
package com.example.tiffinbox.Seller.Model;

public class AddRecipe {
    private String imageURL;
    private String price;
    private String desc;

    public AddRecipe() {
    }

    public AddRecipe(String imageURL, String price, String desc) {
        this.imageURL = imageURL;
        this.price = price;
        this.desc = desc;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getImageURL() {
        return imageURL;
    }

    public void setImageURL(String imageURL) {
        this.imageURL = imageURL;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }
}

File: TiffinBox/app/src/main/java/com/tiff/tiffinbox/Seller/Model/AddRecipe.kt
package com.tiff.tiffinbox.Seller.Model

class AddRecipe {
    var imageURL: String? = null
    var price: String? = null
    var desc: String? = null

    constructor() {}
    constructor(imageURL: String?, price: String?, desc: String?) {
        this.imageURL = imageURL
        this.price = price
        this.desc = desc
    }
}